Classes/MOC/Varnish/Service/ContentCacheFlusherService.php

```diff
@@ -30,11 +30,22 @@ class ContentCacheFlusherService {
 	protected $domainsToFlush = array();
 
 	/**
+	 * @var array
+	 */
+	protected $settings = array();
+
+	public function injectSettings(array $settings) {
+			$this->settings = $settings;
+	}
```

Comment:
instead of injecting all settings, please use the @Configuration\Inject like
````
    /**
     * @Flow\InjectConfiguration("enabled")
     * @var boolean
     */
    protected $enabled;
````